src/openrct2/Context.cpp

```diff
@@ -104,6 +107,9 @@ namespace OpenRCT2
         // Remove this when GetContext() is no longer called so that
         // multiple instances can be created in parallel
         static Context * Instance;
+        static void RunFixedFrameFunc() {
+            Instance->RunFixedFrame();
+        }
```

Comment:
I thought we were trying to discourage singletons?

Reply:
I can see this being only a temporary thing, with the way the comment words it. Sometimes you have to start small, especially in a script system, (that is, if this is part of the script / plugin support), you want to make sure it at least functions, and then work your way up to not using singletons.